spec/acceptance/agent_spec.rb

```diff
@@ -11,10 +13,11 @@ def agent_supported(version)
   describe "zabbix::agent class with zabbix_version #{version}", if: agent_supported(version) do
     it 'works idempotently with no errors' do
       pp = <<-EOS
+      include apt
```

Comment:
Why is including apt needed? It shouldn't be. There's this:
https://github.com/voxpupuli/puppet-zabbix/blob/87e09e3c31c28f0d6c58590f727b92ad5d1c8326/manifests/repo.pp#L98-L104
However, there's also https://github.com/puppetlabs/puppetlabs-apt/blob/04b6a898f8e6f733e1b4f62d3c1a1c631b10253c/manifests/source.pp#L71

I don't see why the repo class shouldn't unconditionally include it. It's not a literal class statement so it shouldn't be an issue. However, it defaults to true anyway so this should be redundant.

```suggestion
```

(repeated comment)